releases/net471/KnownIssues/521954 - BCL CultureAwareComparer with ignore casing on serialized on previous versions of .NET do not correctly deserialize on .NET 4.7.1.md

```diff
@@ -0,0 +1,22 @@
+# CultureAwareComparer with ignore casing serialized on previous versions of .NET Framework do not correctly deserialize on .NET Framework 4.7.1 
+
+## Symptoms
+
+When deserialzing a CultureAwareCompare that was serialized using ignore case on a previous version of .NET Framework on a machine running .NET Framework 4.7.1 the comparer will become a case sensitive comparer.
```

Comment:
Since there is no CultureAwareCompare type, it seems best to change this to "culture-aware comparer"
comma after 4.7.1
the comparer will become a case sensitive comparer --> the comparer becomes case-sensitive.